Platform/obc/hal/comms_hal.c

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (C) 2015  University of Alberta
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ */
+/**
+ * @file comms_hal.c
+ * @author Arash Yazdani
+ * @date 2020-10-01
+ */
+
+/*Once the hardware is connected, toggle this Macro*/
+#define Stubbed = 1;
```

Comment:
Make this a pre-defined macro in the CCS settings to clean this up a little bit

Reply:
I'd say let's take out `stubbed` completely, and do the compiler macro. Make sure that the compiler macro is specific to the module, i.e. `SBAND_IS_STUBBED` or something. Yeah, that would have to be a change in the `ex2_obc_software` code.

Reply:
Or the project file I guess

Reply:
Yeah, I was talking about how the compiler macro would look like. Got it.